components/Modal/CreateBlueprint.js

```diff
@@ -105,26 +108,45 @@ class CreateBlueprintModal extends React.Component {
   }
 
   dismissErrors() {
-    this.setState({ errorNameEmpty: false, errorNameDuplicate: false, errorInline: false });
+    this.setState({
+      errorNameEmpty: false,
+      errorNameDuplicate: false,
+      errorNameSpace: false,
+      errorInline: false,
+      errorNameInvalid: false,
+      errorNameInvalidChar: ""
+    });
   }
 
-  handleErrors(blueprintName) {
-    this.handleErrorDuplicate(blueprintName);
-    this.handleErrorName(blueprintName);
-  }
+  handleErrorNameInvalid(blueprintName) {
+    if (blueprintName.length === 0 && this.state.checkErrors) {
+      this.setState({ errorNameEmpty: true });
+    } else {
+      const invalidCharsRegex = /[^a-zA-Z0-9_,.:+*-]/;
```

Comment:
Can a name contain + * , : 
I find it weird but if the backend allows it I am totally fine.

Reply:
I agree that it is a little strange but the backend seems to allow those characters.